apps/meteor/ee/client/omnichannel/components/RoomActivityIcon/index.tsx

```diff
@@ -0,0 +1,22 @@
+import type { IOmnichannelRoom } from '@rocket.chat/core-typings';
+import { Icon } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+import { useIsRoomActive } from '../../../../../client/hooks/omnichannel/useIsRoomActive';
+import { useOmnichannel } from '../../../../../client/hooks/omnichannel/useOmnichannel';
+
+type RoomActivityIconProps = {
+	room: IOmnichannelRoom;
+};
+
+export const RoomActivityIcon = ({ room }: RoomActivityIconProps): ReactElement | null => {
+	const t = useTranslation();
+	const { isEnterprise } = useOmnichannel();
+	const isRoomActive = useIsRoomActive(room);
```

Comment:
thats the reason I dont like the 'active' term,  it may exists another reasons to make a room inactive, nothing related to MAC counters